insights/client/apps/ansible/__init__.py

```diff
@@ -1,6 +1,25 @@
-
+import os
+import sys
 __all__ = ("verify_playbook")
 
+####  PLAYBOOK VALIDATION ERROR MESSAGE ####
+class PlaybookValidationError(Exception):
+    """
+    Exception raised when playbook validation fails
+
+    Attributes:
+        playbook -- stringified playbook yaml from stdin
+        message -- explanation of why validation failed
+    """
+
+    def __init__(self, playbook, message = " #### PLAYBOOK VALIDATION FAILED ####: ", code = -1):
```

Comment:
Is there a certain use case you are thinking of where the exception needs to keep track of the playbook and code?  I was thinking of just passing the error message into the exception.  I'm ok with passing in the playbook and code if you think we'll need them later on.

Reply:
Hey!  Yeah so when I was putting that together I figured that it would be helpful to know exactly what the playbook was that failed.  However I'm totally cool with just using the message!